Только в linux-2.4.32: arch
Только в linux-2.4.32: .config
Только в linux-2.4.32: .config.old
Только в linux-2.4.32: .configured
Только в linux-2.4.32: COPYING
Только в linux-2.4.32: CREDITS
Только в linux-2.4.32: crypto
Только в linux-2.4.32: .depend
Только в linux-2.4.32: .depend_done
Только в linux-2.4.32: Documentation
Только в linux-2.4.32/drivers: acorn
Только в linux-2.4.32/drivers: acpi
Только в linux-2.4.32/drivers: atm
Только в linux-2.4.32/drivers: block
Только в linux-2.4.32/drivers: bluetooth
Только в linux-2.4.32/drivers: cdrom
Только в linux-2.4.32/drivers: char
Только в linux-2.4.32/drivers: .depend
Только в linux-2.4.32/drivers: dio
Только в linux-2.4.32/drivers: fc4
Только в linux-2.4.32/drivers: gsc
Только в linux-2.4.32/drivers: hil
Только в linux-2.4.32/drivers: hotplug
Только в linux-2.4.32/drivers: i2c
Только в linux-2.4.32/drivers: ide
Только в linux-2.4.32/drivers: ieee1394
Только в linux-2.4.32/drivers: input
Только в linux-2.4.32/drivers: isdn
Только в linux-2.4.32/drivers: macintosh
Только в linux-2.4.32/drivers: Makefile
Только в linux-2.4.32/drivers: md
Только в linux-2.4.32/drivers: media
Только в linux-2.4.32/drivers: message
Только в linux-2.4.32/drivers: misc
Только в linux-2.4.32/drivers: mtd
Только в linux-2.4.32/drivers/net: 3c501.c
Только в linux-2.4.32/drivers/net: 3c503.c
Только в linux-2.4.32/drivers/net: 3c503.h
Только в linux-2.4.32/drivers/net: 3c505.c
Только в linux-2.4.32/drivers/net: 3c505.h
Только в linux-2.4.32/drivers/net: 3c507.c
Только в linux-2.4.32/drivers/net: 3c509.c
Только в linux-2.4.32/drivers/net: 3c515.c
Только в linux-2.4.32/drivers/net: 3c523.c
Только в linux-2.4.32/drivers/net: 3c523.h
Только в linux-2.4.32/drivers/net: 3c527.c
Только в linux-2.4.32/drivers/net: 3c527.h
Только в linux-2.4.32/drivers/net: 3c59x.c
Только в linux-2.4.32/drivers/net: 7990.c
Только в linux-2.4.32/drivers/net: 7990.h
Только в linux-2.4.32/drivers/net: 8139cp.c
Только в linux-2.4.32/drivers/net: 8139too.c
Только в linux-2.4.32/drivers/net: 82596.c
Только в linux-2.4.32/drivers/net: 8390.c
Только в linux-2.4.32/drivers/net: 8390.h
Только в linux-2.4.32/drivers/net: a2065.c
Только в linux-2.4.32/drivers/net: a2065.h
Только в linux-2.4.32/drivers/net: ac3200.c
Только в linux-2.4.32/drivers/net: acenic.c
Только в linux-2.4.32/drivers/net: acenic_firmware.h
Только в linux-2.4.32/drivers/net: acenic.h
Только в linux-2.4.32/drivers/net: adm5120sw.c
diff -ur linux-2.4.32/drivers/net/adm5120sw.h linux-2.4.32-new/drivers/net/adm5120sw.h
--- linux-2.4.32/drivers/net/adm5120sw.h	2006-02-28 12:55:46.000000000 +0200
+++ linux-2.4.32-new/drivers/net/adm5120sw.h	2006-04-19 09:20:44.000000000 +0300
@@ -71,7 +71,7 @@
 //Common
 #define OWN_BIT				0x80000000
 #define END_OF_RING			0x10000000
-#define BUF_ADDR_MASK			0x00FFFFFF
+#define BUF_ADDR_MASK			0x01FFFFFF
 #define BUF2_EN				0x80000000
 
 #define BUF1_LEN_MASK			0x000007FF
Только в linux-2.4.32/drivers/net: .adm5120sw.h.swp
Только в linux-2.4.32/drivers/net: adm5120sw.o
Только в linux-2.4.32/drivers/net: .adm5120sw.o.flags
Только в linux-2.4.32/drivers/net: aironet4500_card.c
Только в linux-2.4.32/drivers/net: aironet4500_core.c
Только в linux-2.4.32/drivers/net: aironet4500.h
Только в linux-2.4.32/drivers/net: aironet4500_proc.c
Только в linux-2.4.32/drivers/net: aironet4500_rid.c
Только в linux-2.4.32/drivers/net: am79c961a.c
Только в linux-2.4.32/drivers/net: am79c961a.h
Только в linux-2.4.32/drivers/net: amd8111e.c
Только в linux-2.4.32/drivers/net: amd8111e.h
Только в linux-2.4.32/drivers/net: apne.c
Только в linux-2.4.32/drivers/net: appletalk
Только в linux-2.4.32/drivers/net: arcnet
Только в linux-2.4.32/drivers/net: ariadne.c
Только в linux-2.4.32/drivers/net: ariadne.h
Только в linux-2.4.32/drivers/net: arlan.c
Только в linux-2.4.32/drivers/net: arlan.h
Только в linux-2.4.32/drivers/net: arlan-proc.c
Только в linux-2.4.32/drivers/net: at1700.c
Только в linux-2.4.32/drivers/net: atari_bionet.c
Только в linux-2.4.32/drivers/net: atarilance.c
Только в linux-2.4.32/drivers/net: atari_pamsnet.c
Только в linux-2.4.32/drivers/net: atp.c
Только в linux-2.4.32/drivers/net: atp.h
Только в linux-2.4.32/drivers/net: au1000_eth.c
Только в linux-2.4.32/drivers/net: au1000_eth.h
Только в linux-2.4.32/drivers/net: auto_irq.c
Только в linux-2.4.32/drivers/net: auto_irq.o
Только в linux-2.4.32/drivers/net: .auto_irq.o.flags
Только в linux-2.4.32/drivers/net: b44.c
Только в linux-2.4.32/drivers/net: b44.h
Только в linux-2.4.32/drivers/net: bagetlance.c
Только в linux-2.4.32/drivers/net: big_sur_ge.c
Только в linux-2.4.32/drivers/net: big_sur_ge.h
Только в linux-2.4.32/drivers/net: bmac.c
Только в linux-2.4.32/drivers/net: bmac.h
Только в linux-2.4.32/drivers/net: bonding
Только в linux-2.4.32/drivers/net: bsd_comp.c
Только в linux-2.4.32/drivers/net: bsd_comp.o
Только в linux-2.4.32/drivers/net: .bsd_comp.o.flags
Только в linux-2.4.32/drivers/net: cirrus.c
Только в linux-2.4.32/drivers/net: cirrus.h
Только в linux-2.4.32/drivers/net: Config.in
Только в linux-2.4.32/drivers/net: cs89x0.c
Только в linux-2.4.32/drivers/net: cs89x0.h
Только в linux-2.4.32/drivers/net: de4x5.c
Только в linux-2.4.32/drivers/net: de4x5.h
Только в linux-2.4.32/drivers/net: de600.c
Только в linux-2.4.32/drivers/net: de620.c
Только в linux-2.4.32/drivers/net: de620.h
Только в linux-2.4.32/drivers/net: declance.c
Только в linux-2.4.32/drivers/net: defxx.c
Только в linux-2.4.32/drivers/net: defxx.h
Только в linux-2.4.32/drivers/net: depca.c
Только в linux-2.4.32/drivers/net: depca.h
Только в linux-2.4.32/drivers/net: .depend
Только в linux-2.4.32/drivers/net: dgrs_asstruct.h
Только в linux-2.4.32/drivers/net: dgrs_bcomm.h
Только в linux-2.4.32/drivers/net: dgrs.c
Только в linux-2.4.32/drivers/net: dgrs_es4h.h
Только в linux-2.4.32/drivers/net: dgrs_ether.h
Только в linux-2.4.32/drivers/net: dgrs_firmware.c
Только в linux-2.4.32/drivers/net: dgrs.h
Только в linux-2.4.32/drivers/net: dgrs_i82596.h
Только в linux-2.4.32/drivers/net: dgrs_plx9060.h
Только в linux-2.4.32/drivers/net: dl2k.c
Только в linux-2.4.32/drivers/net: dl2k.h
Только в linux-2.4.32/drivers/net: dmfe.c
Только в linux-2.4.32/drivers/net: dummy.c
Только в linux-2.4.32/drivers/net: dummy.o
Только в linux-2.4.32/drivers/net: .dummy.o.flags
Только в linux-2.4.32/drivers/net: e100
Только в linux-2.4.32/drivers/net: e1000
Только в linux-2.4.32/drivers/net: e2100.c
Только в linux-2.4.32/drivers/net: eepro100.c
Только в linux-2.4.32/drivers/net: eepro.c
Только в linux-2.4.32/drivers/net: eexpress.c
Только в linux-2.4.32/drivers/net: eexpress.h
Только в linux-2.4.32/drivers/net: epic100.c
Только в linux-2.4.32/drivers/net: eql.c
Только в linux-2.4.32/drivers/net: eql.o
Только в linux-2.4.32/drivers/net: .eql.o.flags
Только в linux-2.4.32/drivers/net: es3210.c
Только в linux-2.4.32/drivers/net: eth16i.c
Только в linux-2.4.32/drivers/net: ether00.c
Только в linux-2.4.32/drivers/net: ethertap.c
Только в linux-2.4.32/drivers/net: ewrk3.c
Только в linux-2.4.32/drivers/net: ewrk3.h
Только в linux-2.4.32/drivers/net: fc
Только в linux-2.4.32/drivers/net: fealnx.c
Только в linux-2.4.32/drivers/net: fmv18x.c
Только в linux-2.4.32/drivers/net: forcedeth.c
Только в linux-2.4.32/drivers/net: gmac.c
Только в linux-2.4.32/drivers/net: gmac.h
Только в linux-2.4.32/drivers/net: gt64240eth.c
Только в linux-2.4.32/drivers/net: gt64240eth.h
Только в linux-2.4.32/drivers/net: gt96100eth.c
Только в linux-2.4.32/drivers/net: gt96100eth.h
Только в linux-2.4.32/drivers/net: hamachi.c
Только в linux-2.4.32/drivers/net: hamradio
Только в linux-2.4.32/drivers/net: hp100.c
Только в linux-2.4.32/drivers/net: hp100.h
Только в linux-2.4.32/drivers/net: hp.c
Только в linux-2.4.32/drivers/net: hplance.c
Только в linux-2.4.32/drivers/net: hplance.h
Только в linux-2.4.32/drivers/net: hp-plus.c
Только в linux-2.4.32/drivers/net: hydra.c
Только в linux-2.4.32/drivers/net: hydra.h
Только в linux-2.4.32/drivers/net: i82586.h
Только в linux-2.4.32/drivers/net: ibm_emac
Только в linux-2.4.32/drivers/net: ibmlana.c
Только в linux-2.4.32/drivers/net: ibmlana.h
Только в linux-2.4.32/drivers/net: ibmveth.c
Только в linux-2.4.32/drivers/net: ibmveth.h
Только в linux-2.4.32/drivers/net: imq.c
Только в linux-2.4.32/drivers/net: imq.o
Только в linux-2.4.32/drivers/net: .imq.o.flags
Только в linux-2.4.32/drivers/net: ioc3-eth.c
Только в linux-2.4.32/drivers/net: irda
Только в linux-2.4.32/drivers/net: isa-skeleton.c
Только в linux-2.4.32/drivers/net: jazzsonic.c
Только в linux-2.4.32/drivers/net: lance.c
Только в linux-2.4.32/drivers/net: lasi_82596.c
Только в linux-2.4.32/drivers/net: LICENSE.SRC
Только в linux-2.4.32/drivers/net: lne390.c
Только в linux-2.4.32/drivers/net: loopback.c
Только в linux-2.4.32/drivers/net: loopback.o
Только в linux-2.4.32/drivers/net: .loopback.o.flags
Только в linux-2.4.32/drivers/net: lp486e.c
Только в linux-2.4.32/drivers/net: mac8390.c
Только в linux-2.4.32/drivers/net: mac89x0.c
Только в linux-2.4.32/drivers/net: mace.c
Только в linux-2.4.32/drivers/net: mace.h
Только в linux-2.4.32/drivers/net: macmace.c
Только в linux-2.4.32/drivers/net: macsonic.c
Только в linux-2.4.32/drivers/net: Makefile
Только в linux-2.4.32/drivers/net: Makefile.lib
Только в linux-2.4.32/drivers/net: mii.c
Только в linux-2.4.32/drivers/net: mv64340_eth.c
Только в linux-2.4.32/drivers/net: mv64340_eth.h
Только в linux-2.4.32/drivers/net: mvme147.c
Только в linux-2.4.32/drivers/net: myri_code.h
Только в linux-2.4.32/drivers/net: myri_sbus.c
Только в linux-2.4.32/drivers/net: myri_sbus.h
Только в linux-2.4.32/drivers/net: natsemi.c
Только в linux-2.4.32/drivers/net: ne2.c
Только в linux-2.4.32/drivers/net: ne2k-pci.c
Только в linux-2.4.32/drivers/net: ne3210.c
Только в linux-2.4.32/drivers/net: ne.c
Только в linux-2.4.32/drivers/net: net_init.c
Только в linux-2.4.32/drivers/net: net_init.o
Только в linux-2.4.32/drivers/net: .net_init.o.flags
Только в linux-2.4.32/drivers/net: net.o
Только в linux-2.4.32/drivers/net: .net.o.flags
Только в linux-2.4.32/drivers/net: ni5010.c
Только в linux-2.4.32/drivers/net: ni5010.h
Только в linux-2.4.32/drivers/net: ni52.c
Только в linux-2.4.32/drivers/net: ni52.h
Только в linux-2.4.32/drivers/net: ni65.c
Только в linux-2.4.32/drivers/net: ni65.h
Только в linux-2.4.32/drivers/net: ns83820.c
Только в linux-2.4.32/drivers/net: oaknet.c
Только в linux-2.4.32/drivers/net: pci-skeleton.c
Только в linux-2.4.32/drivers/net: pcmcia
Только в linux-2.4.32/drivers/net: pcnet32.c
Только в linux-2.4.32/drivers/net: plip.c
Только в linux-2.4.32/drivers/net: ppp_async.c
Только в linux-2.4.32/drivers/net: ppp_async.o
Только в linux-2.4.32/drivers/net: .ppp_async.o.flags
Только в linux-2.4.32/drivers/net: ppp_deflate.c
Только в linux-2.4.32/drivers/net: ppp_deflate.o
Только в linux-2.4.32/drivers/net: .ppp_deflate.o.flags
Только в linux-2.4.32/drivers/net: ppp_generic.c
Только в linux-2.4.32/drivers/net: ppp_generic.o
Только в linux-2.4.32/drivers/net: .ppp_generic.o.flags
Только в linux-2.4.32/drivers/net: ppp_mppe_mppc.c
Только в linux-2.4.32/drivers/net: ppp_mppe_mppc.o
Только в linux-2.4.32/drivers/net: .ppp_mppe_mppc.o.flags
Только в linux-2.4.32/drivers/net: pppoe.c
Только в linux-2.4.32/drivers/net: pppoe.o
Только в linux-2.4.32/drivers/net: .pppoe.o.flags
Только в linux-2.4.32/drivers/net: pppox.c
Только в linux-2.4.32/drivers/net: pppox.o
Только в linux-2.4.32/drivers/net: .pppox.o.flags
Только в linux-2.4.32/drivers/net: ppp_synctty.c
Только в linux-2.4.32/drivers/net: ptifddi_asm.h
Только в linux-2.4.32/drivers/net: ptifddi.c
Только в linux-2.4.32/drivers/net: ptifddi.h
Только в linux-2.4.32/drivers/net: r8169.c
Только в linux-2.4.32/drivers/net: rcif.h
Только в linux-2.4.32/drivers/net: rclanmtl.c
Только в linux-2.4.32/drivers/net: rclanmtl.h
Только в linux-2.4.32/drivers/net: rcpci45.c
Только в linux-2.4.32/drivers/net: rrunner.c
Только в linux-2.4.32/drivers/net: rrunner.h
Только в linux-2.4.32/drivers/net: saa9730.c
Только в linux-2.4.32/drivers/net: saa9730.h
Только в linux-2.4.32/drivers/net: sb1000.c
Только в linux-2.4.32/drivers/net: sb1250-mac.c
Только в linux-2.4.32/drivers/net: seeq8005.c
Только в linux-2.4.32/drivers/net: seeq8005.h
Только в linux-2.4.32/drivers/net: setup.c
Только в linux-2.4.32/drivers/net: setup.o
Только в linux-2.4.32/drivers/net: .setup.o.flags
Только в linux-2.4.32/drivers/net: sgiseeq.c
Только в linux-2.4.32/drivers/net: sgiseeq.h
Только в linux-2.4.32/drivers/net: shaper.c
Только в linux-2.4.32/drivers/net: shaper.o
Только в linux-2.4.32/drivers/net: .shaper.o.flags
Только в linux-2.4.32/drivers/net: sis900.c
Только в linux-2.4.32/drivers/net: sis900.h
Только в linux-2.4.32/drivers/net: sk98lin
Только в linux-2.4.32/drivers/net: skfp
Только в linux-2.4.32/drivers/net: sk_g16.c
Только в linux-2.4.32/drivers/net: sk_g16.h
Только в linux-2.4.32/drivers/net: sk_mca.c
Только в linux-2.4.32/drivers/net: sk_mca.h
Только в linux-2.4.32/drivers/net: slhc.c
Только в linux-2.4.32/drivers/net: slhc.o
Только в linux-2.4.32/drivers/net: .slhc.o.flags
Только в linux-2.4.32/drivers/net: slip.c
Только в linux-2.4.32/drivers/net: slip.h
Только в linux-2.4.32/drivers/net: slip.o
Только в linux-2.4.32/drivers/net: .slip.o.flags
Только в linux-2.4.32/drivers/net: smc9194.c
Только в linux-2.4.32/drivers/net: smc9194.h
Только в linux-2.4.32/drivers/net: smc-mca.c
Только в linux-2.4.32/drivers/net: smc-mca.h
Только в linux-2.4.32/drivers/net: smc-ultra32.c
Только в linux-2.4.32/drivers/net: smc-ultra.c
Только в linux-2.4.32/drivers/net: sonic.c
Только в linux-2.4.32/drivers/net: sonic.h
Только в linux-2.4.32/drivers/net: Space.c
Только в linux-2.4.32/drivers/net: Space.o
Только в linux-2.4.32/drivers/net: .Space.o.flags
Только в linux-2.4.32/drivers/net: starfire.c
Только в linux-2.4.32/drivers/net: stnic.c
Только в linux-2.4.32/drivers/net: strip.c
Только в linux-2.4.32/drivers/net: sun3_82586.c
Только в linux-2.4.32/drivers/net: sun3_82586.h
Только в linux-2.4.32/drivers/net: sun3lance.c
Только в linux-2.4.32/drivers/net: sunbmac.c
Только в linux-2.4.32/drivers/net: sunbmac.h
Только в linux-2.4.32/drivers/net: sundance.c
Только в linux-2.4.32/drivers/net: sungem.c
Только в linux-2.4.32/drivers/net: sungem.h
Только в linux-2.4.32/drivers/net: sunhme.c
Только в linux-2.4.32/drivers/net: sunhme.h
Только в linux-2.4.32/drivers/net: sunlance.c
Только в linux-2.4.32/drivers/net: sunqe.c
Только в linux-2.4.32/drivers/net: sunqe.h
Только в linux-2.4.32/drivers/net: tc35815.c
Только в linux-2.4.32/drivers/net: tg3.c
Только в linux-2.4.32/drivers/net: tg3.h
Только в linux-2.4.32/drivers/net: titan_ge.c
Только в linux-2.4.32/drivers/net: titan_ge.h
Только в linux-2.4.32/drivers/net: titan_mdio.c
Только в linux-2.4.32/drivers/net: titan_mdio.h
Только в linux-2.4.32/drivers/net: tlan.c
Только в linux-2.4.32/drivers/net: tlan.h
Только в linux-2.4.32/drivers/net: tokenring
Только в linux-2.4.32/drivers/net: tulip
Только в linux-2.4.32/drivers/net: tun.c
Только в linux-2.4.32/drivers/net: tun.o
Только в linux-2.4.32/drivers/net: .tun.o.flags
Только в linux-2.4.32/drivers/net: typhoon.c
Только в linux-2.4.32/drivers/net: typhoon-firmware.h
Только в linux-2.4.32/drivers/net: typhoon.h
Только в linux-2.4.32/drivers/net: via-rhine.c
Только в linux-2.4.32/drivers/net: wan
Только в linux-2.4.32/drivers/net: wavelan.c
Только в linux-2.4.32/drivers/net: wavelan.h
Только в linux-2.4.32/drivers/net: wavelan.p.h
Только в linux-2.4.32/drivers/net: wd.c
Только в linux-2.4.32/drivers/net: winbond-840.c
Только в linux-2.4.32/drivers/net: wireless
Только в linux-2.4.32/drivers/net: yellowfin.c
Только в linux-2.4.32/drivers/net: znet.c
Только в linux-2.4.32/drivers/net: zorro8390.c
Только в linux-2.4.32/drivers: nubus
Только в linux-2.4.32/drivers: parport
Только в linux-2.4.32/drivers: pci
Только в linux-2.4.32/drivers: pcmcia
Только в linux-2.4.32/drivers: pnp
Только в linux-2.4.32/drivers: s390
Только в linux-2.4.32/drivers: sbus
Только в linux-2.4.32/drivers: scsi
Только в linux-2.4.32/drivers: sound
Только в linux-2.4.32/drivers: tc
Только в linux-2.4.32/drivers: telephony
Только в linux-2.4.32/drivers: usb
Только в linux-2.4.32/drivers: video
Только в linux-2.4.32/drivers: zorro
Только в linux-2.4.32: fs
Только в linux-2.4.32: .hdepend
Только в linux-2.4.32: include
Только в linux-2.4.32: init
Только в linux-2.4.32: ipc
Только в linux-2.4.32: kernel
Только в linux-2.4.32: lib
Только в linux-2.4.32: linux-2.4.32
Только в linux-2.4.32: MAINTAINERS
Только в linux-2.4.32: Makefile
Только в linux-2.4.32: mm
Только в linux-2.4.32: .modules_done
Только в linux-2.4.32: net
Только в linux-2.4.32: .patched
Только в linux-2.4.32: README
Только в linux-2.4.32: REPORTING-BUGS
Только в linux-2.4.32: Rules.make
Только в linux-2.4.32: scripts
Только в linux-2.4.32: System.map
Только в linux-2.4.32: .unpacked
Только в linux-2.4.32: .version
Только в linux-2.4.32: vmlinux
Только в linux-2.4.32: vmlinux.bin
Только в linux-2.4.32: vmlinuz
